docs/get-started.md

```diff
@@ -45,6 +45,7 @@ import {APIProvider, Map} from '@vis.gl/react-google-maps';
 const App = () => (
   <APIProvider apiKey={API_KEY}>
     <Map
+      style={{width: '500px', height: '700px'}}
```

Comment:
Let's make it fullscreen :) Also fits better with the use of gestureHandling and disableDefaultUI

```suggestion
      style={{width: '100vw', height: '100vh'}}
```